FogCreek/Glitch-Community
# Fallacious slice: Batch together api requests being made on the server

## Links
https://fallacious-slice.glitch.me/
https://github.com/FogCreek/Glitch-Community-Work/issues/328

## Changes:
All api requests for a similar data type within a quarter second get grouped together and sent as a single request. This will increase request time a little bit but hopefully solve the api timeouts we've been experiencing. This won't be enough for ssr project pages, as the two requests added per project to get the users/teams can't be batched.

## How To Test:
Try loading some pages. Make sure that the title and description in the document head are accurate.

## Feedback I'm looking for:
This adds up to a 250ms delay to page requests, which maybe is too much? You can run `cat .data/log/requests.log` in the community-staging console to see a probably average set of requests.
